domain/form-domain/src/main/java/com/uket/domain/form/dto/DropdownFormDto.java

```diff
@@ -0,0 +1,20 @@
+package com.uket.domain.form.dto;
+
+import com.uket.domain.form.entity.DropdownForm;
+import java.util.List;
+import lombok.Builder;
+
+@Builder
+public record DropdownFormDto(
+         Long id,
+         String question,
+         List<String> items
+) implements FormDto {
+    public static DropdownFormDto from(DropdownForm form) {
+        return DropdownFormDto.builder()
+                .id(form.getId())
+                .question(form.getQuestion())
+                .items(form.getItems())
+                .build();
+    }
```

Comment:
변환로직 때문에 고민이신거라면 요런건 어떨까요?
```suggestion
    public static DropdownFormDto from(Form form) {
        DropdownForm dropDownForm = (DropdownForm) form 
    
        return DropdownFormDto.builder()
                .id(dropDownForm.getId())
                .question(dropDownForm.getQuestion())
                .items(dropDownForm.getItems())
                .build();
    }
```

Reply:
변환 로직 때문에 고민했던 게 맞습니다!
형 변환이 타입의 체크가 필요하고, 다형성으로 대체할 수 있는 부분이라고 생각해서
무조건적으로 형변환에 대해서 부정적인 인식이 있었나 생각해보게 되네요!

Reply:
제가 이야기를 잘 못 드린 것 같은데
SurveyDto에서 각 form들을 Dto로 변환하는 로직에 대한 고민이 있었다는 의미입니다!
각 Form들이 Dto 형태로 변환되어야 하는데 이걸 담아줄 그릇이 있어야 할 것 같아서용...